features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/helpers/splitter.js

```diff
@@ -16,6 +16,41 @@ function isEmpty(obj) {
 
     return true;
 }
+
+function transform(businessOwner){
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5